webplugin/js/app/kommunicate.js

```diff
@@ -931,4 +931,26 @@ $applozic.extend(true, Kommunicate, {
             }
         }
     },
+    getCurrentPosition:() => 
+        new Promise((resolve, reject) => 
+          navigator.geolocation.getCurrentPosition(resolve, reject)
+        ),
+    getUserLocation: async function (){
+      try {
+        
+          const api_key='AIzaSyAGVIsWxU7lkCuoodgI6FGXmDN5J11VJFk';
```

Comment:
there is some variable already in the code where google maps api key is present, check for that to reuse

Reply:
check this this location :Kommunicate-Web-SDK/webplugin/js/app/mck-sidebox-1.0.js